### PR TITLE
Add "Setup Remote Machine" recipe.

### LINKED ISSUE #124: Add "setup remote machine user" recipe.

### PR DESCRIPTION
**Pull request by artem-zinnatullin**
Closes #124.

### CODE REVIEW

File: recipes/SETUP_REMOTE_MACHINE.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+## Recipe: Setup Remote Machine
+
+Bash script that allows easily add new users with ssh keys to the remote machine.
```
Comment (arturdryomov) [inline recipes/SETUP_REMOTE_MACHINE.md]: SSH

File: recipes/SETUP_REMOTE_MACHINE.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+# Set bash as default shell for new user, so you could place env vars and other things into `~/.bashrc`.
+chsh -s /bin/bash "$NEW_USER"
+
+# Set user directory as "working dir".
+pushd "/home/$NEW_USER"
+
+# Setup ssh access.
+mkdir -p .ssh
+chmod 700 .ssh
```
Comment (arturdryomov) [inline recipes/SETUP_REMOTE_MACHINE.md]: Letters please.

**Review by arturdryomov — CHANGES_REQUESTED** [review]

**Comment by artem-zinnatullin** [comment]
Foxed.

File: docs/SETUP_REMOTE.md [hunk 3]
```diff
@@ -1,12 +1,15 @@
 # Remote Machine Setup
 
-## Dependencies
+## Required Dependencies
```
Comment (arturdryomov) [inline docs/SETUP_REMOTE.md]: Just dependencies.

**Review by arturdryomov — CHANGES_REQUESTED** [review]

File: docs/SETUP_REMOTE.md (hunk 3 above)
Comment (artem-zinnatullin) [reply docs/SETUP_REMOTE.md]: Why? Let's be more obvious here, no?

Comment (arturdryomov) [reply docs/SETUP_REMOTE.md]: We don’t have optional dependencies, so from my point of view it is quite obvious that dependencies are required.

**Review by arturdryomov — CHANGES_REQUESTED** [review]

**Review by arturdryomov — APPROVED** [review]

**Pull request merged**